portfolio.html

```diff
@@ -217,22 +359,25 @@ <h6 class="connect">LET'S CONNECT</h6>
             </li>
           </ul>
         </div>
-        <input type="button" class="more" value="GET MY RESUME" >
+        <div class="seeMore">
+          <button class="more">GET MY RESUME</button>
+        </div>
+      </div>
         <ul class="dropdownMenu" >
           <li class="ddm1" class="lng"><p>Languages</p>
           <img src="img/open.png" alt="Open dropdown menu"/></li>
         <ul class="dropdownButtons">
-          <li class="buttonLink">
+          <li class="buttonLink ">
             <img src="img/js.png" alt="javaSrcript icon"/>
-              <p>javaScript</p>
+              <p class="js">JavaScript</p>
           </li>
-          <li class="buttonLink">
+          <li class="buttonLink button2">
             <img src="img/html.png" alt="javaSrcript icon"/>
-              <p>HTML</p>
+              <p class="js">HTML</p>
           </li>
-          <li class="buttonLink">
+          <li class="buttonLink button3">
             <img src="img/css.png" alt="javaSrcript icon"/>
-              <p>CSS</p>
+              <p class="js">Css</p>
           </li>
         </ul>
           <li class="ddm"><p>Frameworks</p>
```

Comment:
- You did a great job implementing the requirements from the previous reviewer however there is a small issue you still need to fix. In the language, skill, and frameworks, it should be better to add the space between the arrow icon and the text as required by the previous reviewer [here](https://github.com/TamaraDimov/Portfolio-project/pull/8#discussion_r1043949324) your design must stick as much as possible to the Figma design.
I can suggest you use the same design as you use in the mobile version for framework and skill :+1:  


## Your design 
![image](https://user-images.githubusercontent.com/95297251/206677853-cf9182f4-6001-4052-8aca-31f62e27696d.png)


## Figma design 

![image](https://user-images.githubusercontent.com/95297251/206677488-6fd6a9ab-f6cd-4405-bc25-4378b5de394e.png)